data/data-pipeline/data_pipeline/tests/sources/child_opportunity_index/test_etl.py

```diff
@@ -59,7 +59,7 @@ def test_init(self, mock_etl, mock_paths):
 
     def test_get_output_file_path(self, mock_etl, mock_paths):
         """Tests the right file name is returned."""
-        etl = self._ETL_CLASS()
+        etl = self._get_instance_of_etl_class()
```

Comment:
@mattbowen-usds and I did this to (1) improve test style and (2) make the monkey-patching of the class to only expect the states in the test fixtures work.